example/contract-schema.ts

```diff
@@ -1,20 +1,45 @@
 import { TezosToolkit } from '@taquito/taquito';
-import { InMemorySigner } from '@taquito/signer';
+import { InMemorySigner, importKey } from '@taquito/signer';
+import Faucet from './faucet-interface';
+import BigNumber from 'bignumber.js';
+
+const {email, password, mnemonic, activation_code} = require("./faucet-default-values.json") as Faucet
 
 
 async function example() {
     const provider = 'https://ithacanet.ecadinfra.com/';
     const signer: any = new InMemorySigner('edskRtmEwZxRzwd1obV9pJzAoLoxXFWTSHbgqpDBRHx1Ktzo5yVuJ37e2R4nzjLnNbxFU4UiBU1iHzAy52pK5YBRpaFwLbByca');
-        const tezos = new TezosToolkit(provider);
+    const tezos = new TezosToolkit(provider);
     tezos.setSignerProvider( signer );
+
+    // additional lender if sender is broke
+    const tezosLender = new TezosToolkit(provider);
+    await importKey(
+        tezosLender,
+        email,
+        password,
+        mnemonic.join(' '),
+        activation_code
+      );
     try {
         console.log("signer pkh:");
         console.log(await signer.publicKeyHash());
         const contract = await tezos.contract.at('KT1ERpZEpgtXni64q87VQd6MxgSMrt6Ek65o');
         console.log("Printing contract methods...");
         console.log(contract.methods);
         console.log("Showing initial storage...");
-        console.log(await contract.storage())
+        console.log(await contract.storage());
+        console.log("checking balance")
+        const balance = await tezos.tz.getBalance(await signer.publicKeyHash());
+        const balanceMin = new BigNumber(100);
+        if (balance < balanceMin) {
+            console.log("topping up funds")
+            console.log("Balance before", balance)
+            const sendMutez = await tezosLender.contract.transfer({ to: await signer.publicKeyHash(), amount: 100 });
+            await sendMutez.confirmation();
+            console.log("Balance after")
```

Comment:
Seems like the intention was to log the balance after, but it is missing in the log

Reply:
👍 thanks for the catch :D